### PR TITLE
Miscellaneous changes

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

File: src/Storages/MergeTree/RequestResponse.cpp [hunk 1]
```diff
@@ -60,20 +63,28 @@ void PartitionReadRequest::serialize(WriteBuffer & out) const
...
+        if (!is_first)
+            out << ", ";
+        out << "[" << begin << ", " << end << ")";
+        is_first = false;
+    }
+
+    return out.str();
 }
 
+
```
Comment (hanfei1991) [inline src/Storages/MergeTree/RequestResponse.cpp:87]: redundant?

File: src/Storages/MergeTree/RequestResponse.cpp [hunk 2]
```diff
@@ -95,14 +106,21 @@ void PartitionReadRequest::deserialize(ReadBuffer & in)
 
 UInt64 PartitionReadRequest::getConsistentHash(size_t buckets) const
 {
-    auto hash = SipHash();
+    SipHash hash;
+
+    hash.update(partition_id.size());
```
Comment (hanfei1991) [inline src/Storages/MergeTree/RequestResponse.cpp:111]: What's the difference after we consider size?

Comment (alexey-milovidov) [reply src/Storages/MergeTree/RequestResponse.cpp:111]: In the previous version, only the string contents were hashed, so some combinations of strings may yield identical hashes:
('a', 'bc')
('ab', 'c')

It is a non-significant change, as this hash allows to have collisions, it is more like a readability change, so people will not stumble upon this pice of code thinking it is incorrect.

Comment (hanfei1991) [reply src/Storages/MergeTree/RequestResponse.cpp:111]: it's ok

**Review by hanfei1991 — APPROVED** [review]

**Pull request merged**